ledger/ledger-api-common/src/main/scala/com/digitalasset/platform/server/api/validation/ErrorFactories.scala

```diff
@@ -390,6 +390,22 @@ class ErrorFactories private (errorCodesVersionSwitcher: ErrorCodesVersionSwitch
     )
   }
 
+  def queueClosed(queueName: String)(implicit
```

Comment:
Can you re-use [this one](https://github.com/digital-asset/daml/blob/main/ledger/ledger-api-common/src/main/scala/com/digitalasset/platform/server/api/validation/ErrorFactories.scala#L53-L65)?

Reply:
We might be able to do it once we remove v1 version of the codes. You will notice that the one you have linked uses ABORTED instead of UNAVAILABLE for a legacy error.

Reply:
Right, but do we care? `NoSuchElementException` was not handled before anyway.

Reply:
I reused the old error